compiler-cli/src/run.rs

```diff
@@ -37,8 +42,17 @@ pub fn command(arguments: Vec<String>, target: Option<Target>, which: Which) ->
 
     // Run the command
     let status = match target.unwrap_or(config.target) {
-        Target::Erlang => run_erlang(&module, arguments),
-        Target::JavaScript => run_javascript(&config, &module, arguments),
+        Target::Erlang => match runtime {
+            Some(r) => Err(Error::InvalidRuntime {
+                target: Target::Erlang,
+                invalid_runtime: r,
+            }),
+            _ => run_erlang(&module, arguments),
+        },
+        Target::JavaScript => match config.javascript.runtime {
+            Runtime::Deno => run_javascript_deno(&config, arguments),
+            Runtime::Node => run_javascript_node(&config, arguments),
+        },
```

Comment:
I'm getting a little bit confused here and I'm not sure if it's because I am unwell, but it looks like the Erlang we use the runtime that is passed in on the command line wall with JavaScript we look at the config? Is that right? I think we want to pay attention to both.

Reply:
Yup you were right, fixed so that it will use the value passed to the CLI before defaulting to the config.